diff --git a/node_modules/obs-websocket-js/types/index.d.ts b/node_modules/obs-websocket-js/types/index.d.ts
index 97584d8..8781ec3 100644
--- a/node_modules/obs-websocket-js/types/index.d.ts
+++ b/node_modules/obs-websocket-js/types/index.d.ts
@@ -1144,5 +1144,6 @@ declare module "obs-websocket-js" {
     ): this;
   }
 
-  export = ObsWebSocket;
+  export default ObsWebSocket;
+  export { RequestMethodsArgsMap }
 }
